importer/AmHy.py

```diff
@@ -0,0 +1,94 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class AmHy(Monument):
+
+    def update_labels(self):
+        name = utils.remove_markup(self.description)
+        self.add_label("hy", name)
+
+    def update_descriptions(self):
+        english = "cultural heritage monument of Armenia"
+        self.add_description("en", english)
+
+    def set_adm_location(self):
```

Comment:
So I just spotted that the list also includes monuments in Nagorno-Karabakh (officially Azerbajdzjan). We therefore need to include a data_file for those communities and possibly manually create a data file for those regions (they have non-ISO iso codes which start with NK).

Additionally we need to set country to Q227 (Azerbajdzjan) for these.

Reply:
Note: All the possible prov_iso are [here](https://gist.github.com/Vesihiisi/ffdbdcdd36abdcc012aa5704c8143399). So making a manual file for the NK ones